matter/si91x/siwx917/BRD4338A/autogen/sli_psa_builtin_config_autogen.h

```diff
@@ -0,0 +1,18 @@
+// This is an autogenerated config file, any changes to this file will be
+// overwritten
+
+#ifndef SLI_PSA_BUILTIN_CONFIG_AUTOGEN_H
+#define SLI_PSA_BUILTIN_CONFIG_AUTOGEN_H
+
+#define MBEDTLS_PSA_BUILTIN_KEY_TYPE_AES 1
```

Comment:
For fwiw, we noticed the `PSA_BUILTIN_` causes code size bloat for efr32 and were not needed. We commented out all of them on efr32 boards in matter support